### DISCUSSION TITLE
i want play new map

### DISCUSSION THREAD

**Discussion by kkooka1** [ihhub/fheroes2#6834] [General]
Please tell me where to download

**Comment by ihhub** [comment]
@kkooka1 , please purchase the full game.

**Reply by zenseii** [reply]
Here is one store where you can buy it https://www.gog.com/en/game/heroes_of_might_and_magic_2_gold_edition

It is often on sale for ~2,50€

**Comment by ghost** [comment]
But he wanna new maps, not the original game.

**Reply by zenseii** [reply]
I understood it as this user only having the Demo version of the game, which only comes with the Broken Alliance map. We will have to wait and see what they respond.

**Comment by ihhub** [comment]
This is out of the scope of this project. The Internet contains a lot of websites with maps.

**Comment by ghost** [comment]
I have a lot of new maps, iam primary map creator, but i have not any hub for share it yet.

**Reply by LeHerosInconnu** [reply]
Hello @Jerry11CIB,

> I have a lot of new maps, iam primary map creator, but i have not any hub for share it yet.

New maps, I love new maps. :)
So you have new maps and you want to share them?
If you want you can contact me in private message @Unknown_Hero on the "Forums of Enroth".
